src/rfbzero/experiment.py

```diff
@@ -720,20 +832,33 @@ def run(
             duration, cell_model, degradation, cls_degradation, ncls_degradation, crossover
         )
 
-        def get_cc_cycle_mode(charge):
+        def get_cc_cycle_mode(charge: bool):
             return _ConstantCurrentCycleMode(
-                charge, cell_model, results, update_concentrations,
+                charge,
+                cell_model,
+                results,
+                update_concentrations,
                 self.current_charge if charge else self.current_discharge,
                 self.voltage_limit_charge if charge else self.voltage_limit_discharge,
                 voltage_limit_capacity_check=False
             )
 
-        def get_cv_cycle_mode(charge, current_estimate, current_lim_cls=None, current_lim_ncls=None):
+        def get_cv_cycle_mode(
+                charge: bool,
+                current_estimate: float,
+                current_lim_cls=None,
+                current_lim_ncls=None
+        ):
```

Comment:
Missing return type annotation? `-> _ConstantVoltageCycleMode`

Reply:
done

Reply:
thanks for the fix, it's now working